### PR TITLE
Join tables

### PR DESCRIPTION
**Pull request by Enoisong**
### In this milestone, I:
- add the required data and join tables.
- prepare complex queries that answers analytical questions.

#### Please see the set of screenshots  for the requirements needed to join multiple tables with a join table

- Who was the last animal seen by William Tatcher?
query1

- How many different animals did Stephanie Mendez see?
query2

- List all vets and their specialties, including vets with no specialties.
query3

- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
query4

- What animal has the most visits to vets?
query6

- Who was Maisy Smith's first visit?
query7

- Details for most recent visit: animal information, vet information, and date of visit.
query8

- How many visits were with a vet that did not specialize in that animal's species?
query9

- What specialty should Maisy Smith consider getting? Look for the species she gets the most. 
query11

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -157,3 +157,72 @@ LEFT JOIN animals a ON o.id = a.owner_id
...
+
+-- What specialty should Maisy Smith consider getting? Look for the species she gets the most.
+vet_clinics=# SELECT s.name, COUNT(*) AS num_visits
+vet_clinics-# FROM visits AS v
+vet_clinics-# INNER JOIN animals AS a ON v.animal_id = a.id
+vet_clinics-# INNER JOIN species AS s ON a.species_id = s.id
+vet_clinics-# WHERE v.vet_id IN (SELECT id FROM vets WHERE name = 'Maisy Smith')
+vet_clinics-# GROUP BY s.name
+vet_clinics-# ORDER BY num_visits DESC
+vet_clinics-# LIMIT 1;
```
Comment (franclobo) [inline queries.sql]: - [ ] Please 🙏🏽 use the same format as the rest of the queries here you used a psql format and the rest are SQL commands it is for professional documentation.

File: queries.sql [hunk 2]
```diff
@@ -157,3 +157,72 @@ LEFT JOIN animals a ON o.id = a.owner_id
...
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+ORDER BY v.visit_date DESC
+LIMIT 1;
+
+-- How many visits were with a vet that did not specialize in that animal's species?
+SELECT COUNT(*) AS number_of_visits_without_specialty
+FROM visits v
+LEFT JOIN specializations sp ON v.vet_id = sp.vet_id AND v.animal_id = sp.species_id
+WHERE sp.vet_id IS NULL;
```
Comment (franclobo) [inline queries.sql]: - [ ] It refers to those vets that have a specialization like William Thatcher who specializes in Pokemon but has had visits from Digimon. I mean not only those vets that don't have specialization. Please 🙏🏽 correct this query to show the required information.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Enoisong, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Correct data ✔️ 
- Correct schema ✔️ 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Enoisong, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- Add a Screenshot of the query results to PR ✔️
- Create queries to answer questions in transactions ✔️
- Necessary data was added to data.sql and schema.sql ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**